datadog.rb

```diff
@@ -47,22 +55,18 @@ def nginx_dest
     "/etc/datadog-agent/conf.d/nginx.d/conf.yaml".p
   end
 
+  def up_to_date?(source, dest)
+    Babushka::Renderable.new(dest).from?(source) && Babushka::Renderable.new(dest).clean?
+  end
+
   met? do
-    "/etc/datadog-agent/datadog.yaml".p.grep(/^api_key: #{datadog_api_key}/) &&
-    "/etc/datadog-agent/datadog.yaml".p.grep(/^use_dogstatsd: yes/) &&
-    "/etc/datadog-agent/datadog.yaml".p.grep(/^dogstatsd_port: 8126/) &&
+    up_to_date?(datadog_src, datadog_dest) &&
     nginx_dest.exists?
   end
 
   meet do
-    sudo %(
```

Comment:
We were substituting lines in the example file, it's much simpler to just have our own config from the start.